src/components/CourseChecklist/CourseChecklist.test.jsx

```diff
@@ -1,13 +1,17 @@
-import { Icon } from '@edx/paragon';
+import { Icon, Hyperlink } from '@edx/paragon';
+import { IntlProvider, FormattedMessage } from 'react-intl';
 import React from 'react';
 
 import CourseChecklist from '.';
 import getFilteredChecklist from '../../utils/CourseChecklist/getFilteredChecklist';
 import getValidatedValue from '../../utils/CourseChecklist/getValidatedValue';
+import messages from './displayMessages';
 import { shallowWithIntl } from '../../utils/i18n/enzymeHelper';
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+
 
 // generating test checklist to avoid relying on actual data
-const testChecklistData = ['a', 'b', 'c', 'd'].reduce(((accumulator, currentValue) => { accumulator.push({ id: currentValue, shortDescription: currentValue, longDescription: currentValue }); return accumulator; }), []);
+const testChecklistData = ['welcomeMessage', 'gradingPolicy', 'certificate', 'courseDates', 'assignmentDeadlines'].reduce(((accumulator, currentValue) => { accumulator.push({ id: currentValue }); return accumulator; }), []);
```

Comment:
I had to use actual valid keys here because the CourseChecklist component only successfully renders if the data includes valid keys.

Reply:
That's a fun function to generate the object.